Sources/Model/Receipt/HyperwalletReceipt.swift

```diff
@@ -127,109 +125,204 @@ public struct HyperwalletReceipt: Decodable, Equatable {
 
     /// The transaction type.
     public enum HyperwalletReceiptType: String, Decodable {
-        // Generic fee types
+        /// Generic fee types
+        ///
+        /// The annual fee
         case annualFee = "ANNUAL_FEE"
+        /// The annual fee refund
         case annualFeeRefund = "ANNUAL_FEE_REFUND"
+        /// The customer service fee
         case customerServiceFee = "CUSTOMER_SERVICE_FEE"
+        /// The customer service fee refund
         case customerServiceFeeRefund = "CUSTOMER_SERVICE_FEE_REFUND"
+        /// The expedited shipping fee
         case expeditedShippingFee = "EXPEDITED_SHIPPING_FEE"
+        /// The generic fee refund
         case genericFeeRefund = "GENERIC_FEE_REFUND"
+        /// The monthly fee
         case monthlyFee = "MONTHLY_FEE"
+        /// The monthly fee refund
         case monthlyFeeRefund = "MONTHLY_FEE_REFUND"
+        /// The payment expiry fee
         case paymentExpiryFee = "PAYMENT_EXPIRY_FEE"
+        /// The payment fee
         case paymentFee = "PAYMENT_FEE"
+        /// The processing fee
         case processingFee = "PROCESSING_FEE"
+        /// The standard shipping fee
         case standardShippingFee = "STANDARD_SHIPPING_FEE"
+        /// The transfer fee
         case transferFee = "TRANSFER_FEE"
-        // Generic payment types
+        /// Generic payment types
```

Comment:
No need to add this in document so just // are fine and remove the extra ///

Reply:
Removed extra /